### PR TITLE
Merge old and new types

### PR DESCRIPTION
**Pull request by theengineear**
This finishes merging the old/core stuff back together. I.e., I cleaned up my mess.

### CODE REVIEW

File: applications/desktop/src/notebook/epics/zeromq-kernels.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ import * as jmp from "jmp";
 
 import type { NewKernelAction } from "@nteract/core/src/actionTypes";
 
-import type { OldKernelInfo, LocalKernelProps } from "@nteract/core/src/state";
+import type { KernelInfo, LocalKernelProps } from "@nteract/core/src/state";
```
Comment (rgbkrk) [inline applications/desktop/src/notebook/epics/zeromq-kernels.js:44]: 👍

File: applications/desktop/src/notebook/index.js [hunk 2]
```diff
@@ -24,16 +24,15 @@ import { state } from "@nteract/core";
 
 const store = configureStore({
   app: state.makeAppRecord({
-    host: state.makeOldDesktopHostRecord(),
+    host: state.makeLocalHostRecord(),
```
Comment (rgbkrk) [inline applications/desktop/src/notebook/index.js:27]: 😄

File: packages/core/src/reducers/index.js [hunk 3]
```diff
@@ -3,7 +3,6 @@ import document from "./document";
 import comms from "./comms";
 import config from "./config";
 import app from "./app";
-import modals from "./modals";
 import core from "./core";
```
Comment (rgbkrk) [inline packages/core/src/reducers/index.js:7]: Where are `modals` now?

Comment (rgbkrk) [reply packages/core/src/reducers/index.js:7]: Ah in entities along with hosts! Hurrah!

**Review by rgbkrk — APPROVED** [review]
As the saying goes -- out with the old, in with the core.

🧘

File: packages/core/src/reducers/index.js (hunk 3 above)
Comment (theengineear) [reply packages/core/src/reducers/index.js:7]: Inside `core` so `core.modals`. ✨One less top-level reducer ✨

**Pull request merged**